src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -1043,6 +1046,21 @@ def print_fields(fields):
                 quoting=csv.QUOTE_NONE,
             )
 
+def extract_assay_expression(args):
+    """
+    Retrieve the selected data or generate SQL to retrieve the data from an expression assay in a dataset or cohort based on provided rules.
+    """
+
+    # Validating input combinations
+    input_validator = ExpressionInputsValidator(args, None)
```

Comment:
We can now just use `err_exit` from `dxpy` here and let it handle the writing of messages to the console.

```suggestion
    input_validator = ExpressionInputsValidator(args, err_exit)
```